### PR TITLE
Add development dependencies to suggests and renv ignore

### PR DESCRIPTION
**Pull request by gerbeldo**
# Description
This PR adds packages used for interactive development, such as devtools, roxygen2 and usethis to the list of packages ignored by renv. This has the advantage of allowing developers to have this packages installed, without adding them to the renv.lock file (this includes their dependencies!). They are not necessary at runtime.

Finally, added these devel packages to the Suggest section of the DESCRIPTION file (as is recommended, both by [renv documentation](https://rstudio.github.io/renv/articles/faq.html#how-should-i-handle-development-dependencies) and the [R packages book](https://r-pkgs.org/description-dependencies.html#run-time-vs-develop-time-dependencies). This allows us to install runtime dependencies as usual, using renv::restore() and additionally install the devel dependencies, using renv::install() (with no arguments, it installs the packages listed in the DESCRIPTION FILE)

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
https://github.com/hms-dbmi-cellenics/pipeline/pull/257
https://github.com/hms-dbmi-cellenics/pipeline/pull/263

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [x] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [x] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [x] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [x] Started end-to-end tests on the latest commit.

### Documentation updates
- [x] Relevant Github READMEs updated **or** no GitHub README updates required.
- [x] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: r/DESCRIPTION [hunk 1]
```diff
@@ -2,25 +2,27 @@ Package: rworker
 Title: What the Package Does (One Line, Title Case)
 Version: 0.0.0.9000
 Authors@R: 
-    person(given = "First",
-           family = "Last",
-           role = c("aut", "cre"),
-           email = "first.last@example.com",
+    person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
```
Comment (ogibson) [inline r/DESCRIPTION]: styler?

Comment (gerbeldo) [reply r/DESCRIPTION]: used [usethis::use_tidy_description()](https://usethis.r-lib.org/reference/tidyverse.html#:~:text=use_tidy_description()%3A%20puts%20fields%20in%20standard%20order%20and%20alphabetises%20dependencies.) and it did this. happy to change it back though.

File: r/DESCRIPTION [hunk 2]
```diff
@@ -2,25 +2,27 @@ Package: rworker
...
            comment = c(ORCID = "YOUR-ORCID-ID"))
```
Comment (gerbeldo) [inline r/DESCRIPTION]:
```suggestion
    person(given = "First",
           family = "Last",
           role = c("aut", "cre"),
           email = "first.last@example.com",
           comment = c(ORCID = "YOUR-ORCID-ID"))
```

Comment (gerbeldo) [inline r/DESCRIPTION]:
```suggestion
        person(given = "First",
           family = "Last",
           role = c("aut", "cre"),
           email = "first.last@example.com",
           comment = c(ORCID = "YOUR-ORCID-ID"))
```

**Review by cosa65 — APPROVED** [review]

**Pull request merged**